### PR TITLE
Update cordova.md

### PR DESCRIPTION
**Pull request by cnryb**

### CODE REVIEW

**Comment by dhei** [comment]
Hi @Zakeelm, please review this PR.

File: docs/sdk/getting-started/cordova.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ As a complete example, for a Apache Cordova project that supports both Android a
...
 
-- `APPCENTER_ANALYTICS_ENABLE_IN_JS` - *(optional, default is false)* Controls whether Analytics is enabled automatically. When `false` Analytics is enabled by default. When `true`, the application must call `Analytics.setEnabled()` (in its JavaScript code) to enable Analytics before sending data App Center. This preference is useful when you want to ask users whether they want to share analytics information before enabling it within the application. [Learn more about sending user events manually.](~/sdk/analytics/cordova.md#wait-for-js-to-enable-app-center-analytics)
+- `APPCENTER_ANALYTICS_ENABLE_IN_JS` - *(optional, default is false)* Controls whether Analytics is enabled automatically. When `false` Analytics is enabled by default. When `true`, the application must call `AppCenter.Analytics.setEnabled(true)` (in its JavaScript code) to enable Analytics before sending data App Center. This preference is useful when you want to ask users whether they want to share analytics information before enabling it within the application. [Learn more about sending user events manually.](~/sdk/analytics/cordova.md#wait-for-js-to-enable-app-center-analytics)
```
Comment (ElektrojungeAtWork) [inline docs/sdk/getting-started/cordova.md]: ~1. "When `false`..." requires a comma after `false` if I'm not mistaken.
2. "before sending data TO App Center."~

**Review by ElektrojungeAtWork — CHANGES_REQUESTED** [review]
See my comments.

**Review by ElektrojungeAtWork — APPROVED** [review]

**Review by dhei — APPROVED** [review]

**Comment by jwargo** [comment]
@cnryb thanks for the update.

**Pull request merged**